front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -31,6 +31,7 @@ import {
   ParticipationContextWrapper,
```

Comment:
Could we change projectCard to projectCardImage in the entire file?

Reply:
The card currently means the whole card, also the part under the image. So this renaming is important to avoid confusion.